doc/how_to_start.rst

```diff
@@ -0,0 +1,22 @@
+..
+  SPDX-FileCopyrightText: 2021 The PyPSA meets Earth authors
+
+  SPDX-License-Identifier: CC-BY-4.0
+
+.. _how_to_start:
+
+##########################################
+How to start?
+##########################################
+
```

Comment:
This seems to be empty

Reply:
Let's remove it for now. I think it's worth to supply a very basic starter kit to demonstrate what the model is capable of. But it seems to be a separate task